src/index.js

```diff
@@ -0,0 +1,127 @@
+const updateCity = () => {
+  const userInput = document.querySelector('#user_input').value;
+  document.getElementById('city_name').innerHTML = userInput;
+};
+
+const createResetListener = () => {
+  const resetButton = document.getElementById('reset');
+  resetButton.addEventListener('click', function(){
+    document.getElementById('city_name').innerHTML = 'My Favorite City 😌'; 
+  })
+};
+
+const createSkyListener = () => {
+  document.getElementById('sky_dropdown').addEventListener('change', function() {
+    if (document.getElementById('sky_dropdown').value === "sunny"){
+      document.getElementById('sky_emojis').innerHTML = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+    }
+    if (document.getElementById('sky_dropdown').value === "cloudy"){
+    document.getElementById('sky_emojis').innerHTML =
+      '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+    }
+    console.log(document.getElementById('sky_dropdown').value);
+    if (document.getElementById('sky_dropdown').value === "rainy"){
+      document.getElementById('sky_emojis').innerHTML = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    }
+    console.log(document.getElementById('sky_dropdown').value);
+    if (document.getElementById('sky_dropdown').value === "snowy"){
+    document.getElementById('sky_emojis').innerHTML = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    }
+  });
+
+};
+
+const createTempListeners = () => {
+  const upArrow = document.querySelector("#raise_temp");
+  const downArrow = document.querySelector('#lower_temp');
+  const tempNum = document.getElementById('temp_display');
+  tempNum.innerHTML = 55;
+  upArrow.addEventListener("click", function(){
+    tempNum.innerHTML = parseInt(tempNum.innerHTML) + 1;
+  });
+  downArrow.addEventListener('click', function () {
+    tempNum.innerHTML = parseInt(tempNum.innerHTML) - 1;
+  });
+};
+
+city_name_header;
+change_widgets;
+const createGroundListener = () => {
+  document.getElementById('temp_display').addEventListener('DOMSubtreeModified', function () {
+    if (document.getElementById('temp_display').innerHTML <= 44) {
```

Comment:
Nice work dynamically implementing these styles. It looks great!

Notice that there is quite a bit of repeated code here. Here are a couple ways you might consider refactoring. 1) Rather than applying the style directly to the element, you could assign a class to each element here, and then use that class to apply to styles. Assign each element that you are dynamically styling to a constant, and then you can use that constant to either directly change the style or assign a class name.